tests/commands/download/test_download_hpo_cmd.py

```diff
@@ -4,19 +4,22 @@
 import tempfile
 
 from scout.commands.download.hpo import hpo as hpo_cmd
+from scout.utils.scout_requests import fetch_hpo_files
 
 
 def test_download_hpo_cmd(mocker, empty_mock_app):
     """Test download hpo command"""
+
     # GIVEN a temporary directory
     mock_app = empty_mock_app
     runner = mock_app.test_cli_runner()
 
-    mocker.patch("scout.commands.download.hpo.print_hpo")
```

Comment:
The previous patch was targeting a function called later than fetch_resource (that is getting the resource from internet)